examples/multilabel_classification/1-quickstart/plot_tutorial_multilabel_classification.py

```diff
@@ -102,16 +102,16 @@
 # Bernstein and Waudby-Smith–Ramdas).
 # The two methods give two different guarantees on the risk:
 #
-# * RCPS: :math:`P(R(\mathcal{T}_{\hat{\lambda}})\leq\alpha)\geq 1-\delta`
-#   where :math:`R(\mathcal{T}_{\hat{\lambda}})`
```

Comment:
That one is broken. You can use this tool (an online reStructured editor) to translate : https://feat.dlup.link/rsted

Reply:
Done!